app/models/db/Metrics.scala

```diff
@@ -24,6 +25,14 @@ object Metric {
   implicit val metricEncoder: Encoder[Metric] = deriveEncoder
 }
 
+sealed trait OriginatingSystem extends EnumEntry
```

Comment:
Yeah I think we should stick with OriginatingSystem and I'll pull this out into the library in my work